post-process-html/src/api-doc-types/ApiDoc.ts

```diff
@@ -0,0 +1,5 @@
+export interface ApiDoc {
+  docType: string;
+  renderedContent: string;
+  vFile: any;
```

Comment:
I think that vFile can be typed now, right?

Reply:
I did not find any types in the `rehype` repo, nor on DefinitelyTyped or have I missed something? How should I proceed?

Reply:
I was looking at this: https://github.com/vfile/vfile/pull/33

Reply:
Have missed that, thanks! :)